<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Call rates converter:</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Source call rates CSV:</string>
    </property>
    <property name="buddy">
     <cstring>sourceEdit</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="sourceEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sourceBrowse">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Destination file:</string>
    </property>
    <property name="buddy">
     <cstring>destEdit</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="destEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="destBrowse">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="bar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>511</width>
      <height>27</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="barLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Task progress:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>75</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>S&amp;tart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>75</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Parsed file info</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
